### ISSUE TITLE
Dependency `vscode-languageclient` bumped VS Code requirement to v1.82

### ISSUE THREAD

**Issue by coolhome** [PowerShell/vscode-powershell#4762]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [_open and closed_ issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.
- [X] I have read the [troubleshooting](https://github.com/PowerShell/vscode-powershell/blob/main/docs/troubleshooting.md) guide.
- [x] I am sure this issue is with the _extension itself_ and does not reproduce in a standalone [PowerShell](https://github.com/PowerShell/PowerShell/issues/new/choose) instance.
- [ ] I have verified that I am using the latest version of Visual Studio Code and the PowerShell extension.
- [ ] If this is a security issue, I have read the [security issue reporting guidance](https://github.com/PowerShell/vscode-powershell/blob/main/SECURITY.md).

### Summary

Terminal never fully loaded. Downgrading back down to 2023.9.2

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.3.6
PSEdition                      Core
GitCommitId                    7.3.6
OS                             Microsoft Windows 10.0.22621
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Visual Studio Code Version

```console
1.81.1
6c3e3dba23e8fadc360aed75ce363ba185c49794
x64
```

### Extension Version

```console
ms-vscode.powershell@2023.9.3
```

### Steps to Reproduce

Install preview extension - however there may be more here I am unable to find.

### Visuals

![image](https://github.com/PowerShell/vscode-powershell/assets/700740/3bc19ee3-ebe7-4bdb-881e-07a96b3eaa5a)
![image](https://github.com/PowerShell/vscode-powershell/assets/700740/d70f389c-3a55-48d9-b812-150db2f3b2b6)

This is where its stuck. Downgrading resolved the issue.

**Comment by andyleejordan** [comment]
Hi,

Could you please provide more [logging and environment information by following these steps](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md#providing-information-about-your-environment)?

Thanks!

**Comment by coolhome** [comment]
@andyleejordan Enabling diagnostic on the Language Server produced the root issue.

```
The language client requires VS Code version ^1.82.0 but received. 1.81.1
```

Upgraded to 1.82.2 and it's working again. I'm surprised there was no alert for this.

**Comment by andyleejordan** [comment]
I...did not know it would bump that requirement! That's extremely good information. I should have looked at https://github.com/PowerShell/vscode-powershell/pull/4757 closer (which I assume has to be the dependency that bumped that up).

**Comment by andyleejordan** [comment]
I had to dig a bit and it is indeed that package (or set of packages) as I see it updated in https://github.com/microsoft/vscode-languageserver-node/pull/1312 but yeah, no mention in changelog. I'll downgrade the package and make a note of that.

**Comment by andyleejordan** [comment]
Thanks again @coolhome. Just released v2023.9.4-preview after verifying the extension would start up with VS Code v1.79 (our minimum requirement). Cheers!
